zadanie1

Frodo1996 opened the pull request.

Frodo1996 committed:
zadanie1

slawekludwiczak reviewed (changes requested):
Przenieś plik registrationForm.html do głównego katalogu projektu z folderu .idea
Użyj klas bootstrapa zamiast samodzielnego definiowania stylów

code review https://javastartbootcamp.slack.com/archives/C04AUFUPDU4/p1681372103143749

Frodo1996 committed:
poprawione zadanie pierwsze.
Zrobiłem dwa formularze:
- formularz numer 1 to mój stworzony pierwotnie i poprawiony tak by jak najbardziej pasował do wzoru załączonego do ćwiczenia
- formularz numer 2 stworzony zgodnie z instrukcjami przy użyciu bootstrap
